examples/src/examples/graphics/area-lights.tsx

```diff
@@ -1,196 +1,197 @@
-import React from 'react';
 import * as pc from '../../../../';
-import { AssetLoader } from '../../app/helpers/loader';
 
 
 class AreaLightsExample {
     static CATEGORY = 'Graphics';
     static NAME = 'Area Lights';
 
-    load() {
-        return <>
-            <AssetLoader name='color' type='texture' url='/static/assets/textures/seaside-rocks01-color.jpg' />
-            <AssetLoader name='normal' type='texture' url='/static/assets/textures/seaside-rocks01-normal.jpg' />
-            <AssetLoader name='gloss' type='texture' url='/static/assets/textures/seaside-rocks01-gloss.jpg' />
-            <AssetLoader name='statue' type='container' url='/static/assets/models/statue.glb' />
-            <AssetLoader name='luts' type='binary' url='/static/assets/binary/area-light-luts.bin' />
-            <AssetLoader name='helipad.dds' type='cubemap' url='/static/assets/cubemaps/helipad.dds' data={{ type: pc.TEXTURETYPE_RGBM }}/>
-        </>;
-    }
 
-    example(canvas: HTMLCanvasElement, assets: any): void {
+    example(canvas: HTMLCanvasElement): void {
 
         // Create the app and start the update loop
         const app = new pc.Application(canvas, {});
 
-        // helper function to create a primitive with shape type, position, scale, color
-        function createPrimitive(primitiveType: string, position: pc.Vec3, scale: pc.Vec3, color: pc.Color, assetManifest: any) {
-
-            // create material of specified color
-            const material = new pc.StandardMaterial();
-            material.diffuse = color;
-            material.shininess = 80;
-            material.useMetalness = true;
+        const assets = {
+            'color': new pc.Asset('color', 'texture', { url: '/static/assets/textures/seaside-rocks01-color.jpg' }),
+            'normal': new pc.Asset('normal', 'texture', { url: '/static/assets/textures/seaside-rocks01-normal.jpg' }),
+            'gloss': new pc.Asset('gloss', 'texture', { url: '/static/assets/textures/seaside-rocks01-gloss.jpg' }),
+            'statue': new pc.Asset('statue', 'container', { url: '/static/assets/models/statue.glb' }),
+            'luts': new pc.Asset('luts', 'binary', { url: '/static/assets/binary/area-light-luts.bin' }),
+            'helipad.dds': new pc.Asset('helipad.dds', 'cubemap', { url: '/static/assets/cubemaps/helipad.dds' }, { type: pc.TEXTURETYPE_RGBM })
+        };
+
+        const assetListLoader = new pc.AssetListLoader(Object.values(assets), app.assets);
```

Comment:
IE11 does not implement Object.values .. so just making sure we have some way to make this work (polyfill or transpilation)

Reply:
Yep this example code is transpiled.